util/cron/nightly

```diff
@@ -2,11 +2,14 @@
 
 # Developers: To test changes with nightly against a clean repo, use this command:
 #
-# cd /tmp && \
+# export OLD_CHPL_HOME=$CHPL_HOME && \
```

Comment:
Why is the `OLD_CHPL_HOME` setting required?

Also can you check that these instructions match what's in `util/cron/README` (and maybe we should delete one set of instructions so we don't have to maintain both)

Reply:
It's required because the `source util/setchplenv.bash` line changes `$CHPL_HOME` to the directory in /tmp. We want to test the modified `nightly` that was in the old `$CHPL_HOME`, not the unmodified `nightly` in the new `$CHPL_HOME`.

Reply:
What's in util/cron/README is less about developers testing changes to the nightly script and more about just running it in general. What's in the comment in nightly is meant to give a one-liner for developers "To test changes with nightly against a clean repo". The README shows to test with -cron instead of -debug, so has slightly different settings.

Reply:
> It's required because the source util/setchplenv.bash line changes $CHPL_HOME to the directory in /tmp. We want to test the modified nightly that was in the old $CHPL_HOME, not the unmodified nightly in the new $CHPL_HOME.

I think the expectation is that you've committed any changes you want to test, but it sounds like what you're trying to do is test some non-committed changes, so that's why you're calling the original nightly?

Reply:
Right, I was testing before committing. Should I remove OLD_CHPL_HOME and add a comment saying that this is to test committed changes?

Reply:
I'm fine with this, just wanted to make sure I understood

Reply:
OK, Thanks!